### PR TITLE
add some conda instructions to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by epwalsh**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,36 +38,47 @@ When you're ready to contribute code to address an open issue, please follow the
...
-    Finally, you'll need to create a Python 3 virtual environment suitable for working on this project. There a number of tools out there that making working with virtual environments easier, but the most direct way is with the [`venv` module](https://docs.python.org/3.7/library/venv.html) in the standard library.
+    Finally, you'll need to create a Python 3 virtual environment suitable for working on this project. There a number of tools out there that making working with virtual environments easier.
+    The most direct way is with the [`venv` module](https://docs.python.org/3.7/library/venv.html) in the standard library, but if you're new to Python or you don't already have a recent Python 3 version installed on your machine,
+    the easiest method is to use [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
+
+    On Mac, for example, you can install Miniconda with [Homebrew](https://brew.sh/):
+
+        brew install miniconda
+
+    Then you can create and activate a new Python virtual environment by running:
```
Comment (dirkgr) [inline CONTRIBUTING.md:78]:
```suggestion
    Then you can create and activate a new Python environment by running:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -38,36 +38,47 @@ When you're ready to contribute code to address an open issue, please follow the
...
+
         origin https://github.com/USERNAME/tango.git (fetch)
         origin https://github.com/USERNAME/tango.git (push)
         upstream https://github.com/allenai/tango.git (fetch)
         upstream https://github.com/allenai/tango.git (push)
 
-    Finally, you'll need to create a Python 3 virtual environment suitable for working on this project. There a number of tools out there that making working with virtual environments easier, but the most direct way is with the [`venv` module](https://docs.python.org/3.7/library/venv.html) in the standard library.
+    Finally, you'll need to create a Python 3 virtual environment suitable for working on this project. There a number of tools out there that making working with virtual environments easier.
+    The most direct way is with the [`venv` module](https://docs.python.org/3.7/library/venv.html) in the standard library, but if you're new to Python or you don't already have a recent Python 3 version installed on your machine,
+    the easiest method is to use [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
```
Comment (dirkgr) [inline CONTRIBUTING.md:72]:
```suggestion
    we recommend [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
```

**Review by dirkgr — APPROVED** [review]

**Review by jonborchardt — APPROVED** [review]
where do we find example code to run?
quick start to viz?

**Comment by epwalsh** [comment]
> where do we find example code to run?

- If you're looking for an actual end-to-end real-world example: the [`examples/` directory](https://github.com/allenai/tango/tree/main/examples).
- If you just want to make sure you're not breaking anything, see [Step 4 in CONTRIBUTING.md](https://github.com/allenai/tango/blob/main/CONTRIBUTING.md#making-a-pull-request).
- If you want to go more in-depth, see the [Overview](https://ai2-tango.readthedocs.io/en/latest/overview.html) section of the docs.

> quick start to viz?

The viz doesn't exist on this branch so I'll let you add information about that to your branch, if you want.

**Pull request merged**